lib/wanda_web/controllers/catalog_controller.ex

```diff
@@ -5,13 +5,13 @@ defmodule WandaWeb.CatalogController do
   alias Wanda.Catalog
   alias WandaWeb.Schemas.CatalogResponse
 
-  operation :list_catalog,
+  operation :catalog,
```

Comment:
Should be name this `index` too?

Reply:
index is usually used for resources, this would work if the catalog path was /checks/

I think the catalog is a special case, so I am following what the real world app is doing with the `feed`:
https://github.com/gothinkster/elixir-phoenix-realworld-example-app/blob/8620fb2829703051cdbd032f93be6cd941c4810e/lib/real_world_web/router.ex#L21